packages/js-sdk/src/session/index.ts

```diff
@@ -228,13 +233,19 @@ export class Session extends SessionConnection {
           onStderr,
           onExit,
           envVars = {},
-          rootdir = '',
+          cwd = '',
+          rootDir,
           processID = id(12),
         }: Omit<ProcessOpts, 'timeout'>) => {
-          if (!cmd) {
-            throw new Error('cmd is required')
+          if (!cwd && rootDir) {
+            this.logger.warn?.('The rootDir parameter is deprecated, use cwd instead.')
+            cwd = rootDir
           }
-          this.logger.debug?.(`Starting process "${processID}"`)
+          if (!cwd && this.opts.cwd) {
+            cwd = this.opts.cwd
+          }
+          if (!cmd) throw new Error('cmd is required')
```

Comment:
This is little bit inconsistent. It doesn't work like this for terminal (either we should delete it or keep it maintained), nor in python,